### ISSUE TITLE
Error when parsing int64 or double array

### ISSUE THREAD

**Issue by ityuhui** [kubernetes-client/c#70]
Reported by @clearday4 

the type of supplemental_groups is as below:
```yaml
        "supplementalGroups": {
          "description": "A list of groups applied to the first process run in each container, in addition to the container's primary GID.  If unspecified, no groups will be added to any container.",
          "items": {
            "format": "int64",
            "type": "integer"
          },
          "type": "array"
        },
```
it is an integer array.

We suggest changing it as follows.
In v1_pod_security_context.c
```c
list_addElement(supplemental_groupsList , &supplemental_groups_local->valuedouble);
```
=> 
```c  
double *supplemental_groups_local_values = malloc(sizeof(double));
*supplemental_groups_local_values = supplemental_groups_local->valuedouble;
list_addElement(supplemental_groupsList , supplemental_groups_local_values);
```

**Comment by ityuhui** [comment]
The fix in generator is https://github.com/OpenAPITools/openapi-generator/pull/10124

**Comment by ityuhui** [comment]
Fixed by #73
